astropy/table/table.py

```diff
@@ -207,16 +207,17 @@ class Table:
         Metadata associated with the table.
     copy : bool, optional
         Copy the input data. Default is True.
+
```

Comment:
If things do not work, I think it is completely fine to just add this note directly to the text, without `.. note`